app/bundles/LeadBundle/Translations/en_US/messages.ini

```
@@ -263,6 +263,8 @@ mautic.lead.lead.events.tags="Contact tags"
 mautic.lead.lead.events.tags_descr="Condition based on a contact tags."
 mautic.lead.lead.events.segments="Contact segments"
 mautic.lead.lead.events.segments_descr="Condition based on a contact segments."
+mautic.lead.lead.events.stages="Contact stages"
+mautic.lead.lead.events.stages_descr="Condition based on a contact stages."
```

I'd also like to make obvious for the users what it would mean if they select more than one stage. Can we change this to something like
```suggestion
mautic.lead.lead.events.stages_descr="Condition that the contact belongs to at least one of the selected stages."
```

---

I like it more too